core/src/main/java/org/namstorm/deltaforce/core/DeltaBuilder.java

```diff
@@ -33,6 +33,17 @@
     T apply();
 
     /**
+     * same as apply() but also clear the deltas afterwards
+     * @return object that's been applied to
+     */
+    T applyAndClearDeltas();
+
+    /**
+     * clears out the deltas
+     */
+    void rollback();
```

Comment:
Rename this as clear? In previous method, applyAndClearDeltas(), the clear word is used.